.github/workflows/register-buildpack.yml

```
@@ -0,0 +1,143 @@
+name: Register Buildpack
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  register:
+    env:
+      GITHUB_OWNER: buildpacks
+      INDEX_REPO: registry-index
+      NAMESPACE_REPO: registry-namespaces
```

I would refrain from using `GITHUB_` prefix since GHA uses it for it's provided env vars. I would hate to inadvertently stomp on something they may add in the future.

```suggestion
      GH_REG_OWNER: buildpacks
      GH_REG_INDEX_REPO: registry-index
      GH_REG_NAMESPACE_REPO: registry-namespaces
```

---

Good point!  I like the suggestions.